content/index.ko.md

```diff
@@ -100,8 +101,7 @@ footer: >   # Translate all the words below, including "Date:" and "Editor:". Do
 
 접근성 기능이 한 요소에서 효과적으로 실행된다면, 다른 요소들도 해당 기능을 실행할 수 있을 것입니다. 
 
-{% assign example_url = "/fundamentals/components/examples/#cycle" | relative_url %}
-![실행 사이클을 보여주는 삽화. 자세한 설명은 다음 링크 참고 : {{ example_url }}]({{ "/content-images/wai-components/cycle.png" | relative_url }}){:longdesc="{{example_url}}"}
+{% include image.html src="cycle.png" alt="" %}
 
 - **웹 브라우저, 미디어 플레이어, 보조 기술, 사용자 에이전트**가 접근성 기능을 지원할 때 사용자는 해당 기능을 더 요구할 것이고, 개발자들은 더 쉽게 그들의 **콘텐츠**에  해당 기능을 적용할 수 있을 것입니다.
 - 개발자가 그들의 **콘텐츠**에 접근성 기능을 적용하고 싶을 때 그들의 **사용자 에이전트**에 해당 기능을 더 쉽게 이용할 수 있도록 요구할 수 있습니다. 
```

Comment:
Original version:

> When developers want to implement an accessibility feature in their content, they are more likely to demand that their authoring tool make it easy to implement.

@hwahyeon This needs update: the translation talks about "user agents" rather than "authoring tool".

Reply:
Commit 5c819b4